pyomo/solvers/tests/piecewise_linear/test_examples.py

```diff
@@ -39,57 +40,74 @@ def run_convert2lp(self, name):
     def test_step_lp(self):
         """Test examples/pyomo/piecewise/step.py"""
         self.run_convert2lp('step.py')
-        self.assertFileEqualsBaseline(join(currdir,'unknown.lp'), currdir+'step.lp', tolerance=_LP_diff_tol)
+        self.assertTrue(cmp(join(currdir,'unknown.lp'), currdir+'step.lp'))
```

Comment:
As with above, it would be helpful to include a failure message here (and the 6 uses below).